src/interfaces/package-export-transport.ts

```diff
@@ -0,0 +1,27 @@
+import {StudioComputeNodeDescriptor} from "./package-manager.interfaces";
+
+export interface PackageVersionTransport {
+    id: string;
+    key: string;
+    name: string;
+    version: string;
+}
+
+export interface DependencyTransport {
+    key: string;
+    version: string;
+}
+
+export interface PackageExportTransport {
+    id: string;
+    key: string;
+    name: string;
+    changeDate: string;
+    activatedDraftId: string;
+    workingDraftId: string;
+    flavor: string;
+    version: PackageVersionTransport;
```

Comment:
With the latest [updates](https://github.com/celonis/cloud-package-manager/blob/develop/cloud-package-manager/src/main/java/cloud/celonis/packages/manager/exportimport/PackageExportTransport.java#L25) "version" is of type String.